qa/vagrant/src/main/java/org/elasticsearch/packaging/test/ArchiveTestCase.java

```diff
@@ -96,11 +96,9 @@ public void test10Install() {
     public void test20PluginsListWithNoPlugins() {
         assumeThat(installation, is(notNullValue()));
 
-        // todo here
+        final Installation.Executables bin = installation.executables();
         final Shell sh = new Shell();
-        final Result r = Platforms.WINDOWS
-            ? sh.powershell(installation.bin("elasticsearch-plugin.bat") + " list")
-            : sh.bash(installation.bin("elasticsearch-plugin") + " list");
+        final Result r = sh.run(bin.elasticsearchPlugin + " list");
```

Comment:
>This will also need a sh.run(String executable, String... args) that joins args with a space

I considered doing this and decided against it because all the commands are run using `"bash", "-c", "[everything passed to Shell.run]"`, and making it look like it handles arguments separately would be trappy (i.e. this makes it clear that Shell does no quoting/escaping for you)